### PR TITLE
Icon Updates

### PR DESCRIPTION
**Pull request by Horchen154**
Ceiling LIght, dimmer, light switch, lightbulb, outlet, power plug added

Fixes # .

#### Changes proposed in this Pull Request:
- 
- 
-

### CODE REVIEW

**Comment by huayunh** [comment]
@Horchen154 make sure you have that "use artboard" thing checked when you do Export > Export as

----
Note to myself -- TODO
- update meta.json
-  run yarn build in a separate branch

**Comment by joebochill** [comment]
Icons shapes and artboard are looking good, but there's still the issue of the fill property. And can we make LightBulb into two words? Most references online and in publications use this format (even though Merriam-Webster uses one)...my spellchecker complains about it.

image

File: design/Lighting/ceiling_fan.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>ceiling_fan</title><path d="M8,9h3V4h2V9h3l4,8H4Zm6,9a2,2,0,0,1-4,0Z" fill="#727e84"/></svg>
```
Comment (joebochill) [inline design/Lighting/ceiling_fan.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>ceiling_fan</title><path d="M8,9h3V4h2V9h3l4,8H4Zm6,9a2,2,0,0,1-4,0Z"/></svg>
```

File: design/Lighting/dimmer.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>dimmer</title><path d="M13,5H11v7H9v2h6V12H13Z" fill="#727e84"/><path d="M11,15h2v4H11Z" fill="#727e84"/><path d="M5,3A2,2,0,0,1,7,1H17a2,2,0,0,1,2,2V21a2,2,0,0,1-2,2H7a2,2,0,0,1-2-2ZM7,3H17V21H7Z" fill="#727e84" fill-rule="evenodd"/></svg>
```
Comment (joebochill) [inline design/Lighting/dimmer.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>dimmer</title><path d="M13,5H11v7H9v2h6V12H13Z" fill="#727e84"/><path d="M11,15h2v4H11Z" fill="#727e84"/><path d="M5,3A2,2,0,0,1,7,1H17a2,2,0,0,1,2,2V21a2,2,0,0,1-2,2H7a2,2,0,0,1-2-2ZM7,3H17V21H7Z" fill-rule="evenodd"/></svg>
```

File: design/Lighting/light_switch.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>light_switch</title><path d="M18.41,1.59A2,2,0,0,0,17,1H7A2,2,0,0,0,5,3V21a2,2,0,0,0,2,2H17a2,2,0,0,0,2-2V3A2,2,0,0,0,18.41,1.59ZM17,21H7V3H17Z" fill="#727e84"/><path d="M15,6H9V18h6ZM13,16H11V13h2Zm0-5H11V8h2Z" fill="#727e84"/></svg>
```
Comment (joebochill) [inline design/Lighting/light_switch.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>light_switch</title><path d="M18.41,1.59A2,2,0,0,0,17,1H7A2,2,0,0,0,5,3V21a2,2,0,0,0,2,2H17a2,2,0,0,0,2-2V3A2,2,0,0,0,18.41,1.59ZM17,21H7V3H17Z" fill="#727e84"/><path d="M15,6H9V18h6ZM13,16H11V13h2Zm0-5H11V8h2Z" /></svg>
```

File: design/Lighting/lightbulb.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>lightbulb</title><path d="M12,2A7,7,0,0,0,8,14.74V17a1,1,0,0,0,1,1h6a1,1,0,0,0,1-1V14.74A7,7,0,0,0,12,2ZM9,21a1,1,0,0,0,1,1h4a1,1,0,0,0,1-1V20H9Z" fill="#727e84"/></svg>
```
Comment (joebochill) [inline design/Lighting/lightbulb.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>lightbulb</title><path d="M12,2A7,7,0,0,0,8,14.74V17a1,1,0,0,0,1,1h6a1,1,0,0,0,1-1V14.74A7,7,0,0,0,12,2ZM9,21a1,1,0,0,0,1,1h4a1,1,0,0,0,1-1V20H9Z" /></svg>
```

File: design/Lighting/outlet.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>outlet</title><path d="M8,7h2v5H8ZM4.22,2H19.78A2.23,2.23,0,0,1,22,4.22V19.78A2.22,2.22,0,0,1,19.78,22H4.22A2.23,2.23,0,0,1,2,19.78V4.22A2.22,2.22,0,0,1,4.22,2ZM12,4a8,8,0,1,0,5.66,2.34A8,8,0,0,0,12,4Zm2,3.5h2v4H14Zm-3.5,8.75a1.5,1.5,0,0,1,3,0V17h-3Z" fill="#727e84"/></svg>
```
Comment (joebochill) [inline design/Lighting/outlet.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>outlet</title><path d="M8,7h2v5H8ZM4.22,2H19.78A2.23,2.23,0,0,1,22,4.22V19.78A2.22,2.22,0,0,1,19.78,22H4.22A2.23,2.23,0,0,1,2,19.78V4.22A2.22,2.22,0,0,1,4.22,2ZM12,4a8,8,0,1,0,5.66,2.34A8,8,0,0,0,12,4Zm2,3.5h2v4H14Zm-3.5,8.75a1.5,1.5,0,0,1,3,0V17h-3Z"/></svg>
```

File: design/Lighting/power_plug.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>power_plug</title><path d="M16,7V3H14V7H10V3H8V7A2.15,2.15,0,0,0,6,9v5.5L9.5,18v3h5V18L18,14.5V9A2.15,2.15,0,0,0,16,7Z" fill="#727e84"/></svg>
```
Comment (joebochill) [inline design/Lighting/power_plug.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><title>power_plug</title><path d="M16,7V3H14V7H10V3H8V7A2.15,2.15,0,0,0,6,9v5.5L9.5,18v3h5V18L18,14.5V9A2.15,2.15,0,0,0,16,7Z"/></svg>
```

**Comment by huayunh** [comment]
Looking for a nod on https://github.com/pxblue/icons/pull/85

File: design/META.json [hunk 7]
```diff
@@ -2812,6 +2812,98 @@
...
+                "energy",
+                "power"
+            ],
+            "author": "Eric Atha",
+            "size": 24
+        }
+        ,
+        {
+            "name": "Dimmer",
+            "filename": "dimer.svg",
```
Comment (joebochill) [inline design/META.json:2835]:
```suggestion
            "filename": "dimmer.svg",
```

**Review by huayunh — APPROVED** [review]
Let's go :) 

Time for publishing a new version

**Review by joebochill — APPROVED** [review]
👍

**Pull request merged**